src/unit_tests/config/config_test.py

```diff
@@ -16,6 +16,7 @@ class testConfigVars:
     schema_path = "test_schema_path"
     api_url = "test_api_url"
     oauth_client_id = "test_oauth_client_id"
+    survey_map_url = "https://raw.githubusercontent.com/ONSdigital/sds-schema-definitions/main/mapping/survey_map.json"
```

Comment:
We probably only need a test string here as it is for unit testing whether the config variable can be correctly loaded to environment only

Reply:
@IsaiahLai111 Just changed this to a test value.